spec/textris/phone_formatter_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe Textris::PhoneFormatter do
+  it 'should recognise a 4 digit short code' do
+    expect(described_class.format("4437")).to eq('4437')
+  end
+
+  it 'should recognise a 5 digit short code' do
+    expect(described_class.format("44397")).to eq('44397')
+  end
+
+  it 'should recognise a 6 digit short code' do
+    expect(described_class.format("443975")).to eq('443975')
+  end
+
+  it 'treat non-short code non-phone number numbers as alphamerics' do
+    ['8945', '894', '89', '8', '8945467', '89454678', '894546789'].each do |number|
```

Comment:
These tests are incorrect, alphanumeric sender IDs are required to have at least one letter.

Reply:
This test case was not supposed to check whether it's classified as alphameric, but rather 'treated' as it was like it (so no additional formatting), if I remember correctly :) but yes, this should be replaced with a proper alphamerics tests, I agree.